sfs/mono/source.py

```diff
@@ -169,11 +169,9 @@ def point_modal(omega, x0, n0, grid, L, N=None, deltan=0, c=None):
     L : (3,) array_like
         Dimensionons of the rectangular room.
     N : (3,) array_like or int, optional
-        Combination of modal orders in the three-spatial dimensions to
-        calculate the sound field for or maximum order for all
-        dimensions.  If not given, the maximum modal order is
-        approximately determined and the sound field is computed up to
-        this maximum order.
+        For all three spatial dimensions per dimension maximum order,
+        list of orders used or if not given maximum modal order is
+        approximately determined. A scalar applies to all three dimensions.
```

Comment:
This is a little bit difficult and I would prefer to have the default behavior at the end, so maybe:
```
For all three spatial dimensions per dimension maximum order or
￼list of orders. A scalar applies to all three dimensions. If no
order is provided it is approximately determined. 
```